### DISCUSSION TITLE
QUIC_PERF_COUNTER_PKTS_SUSPECTED_LOST meaning

### DISCUSSION THREAD

**Discussion by akrisfx** [microsoft/msquic#4711] [Q&A] [answered]
in QUIC_PERFORMANCE_COUNTERS we have QUIC_PERF_COUNTER_PKTS_SUSPECTED_LOST
In which direction are they lost relative to the quic server when sending or receiving?

**Comment by nibanks** [comment, marked as answer]
This is in the send direction. This is the count of packets that we suspected were lost (but not yet confirmed).
